### ISSUE TITLE
Include package name?

### ISSUE THREAD

**Issue by IzzySoft** [techlore/plexus#36]
It might be helpful to also record the package name / Playstore link to uniquely identify an app, as  some names are shared by multiple apps (e.g. just try to identify which app is meant by "Music Player"). As this project focuses on apps from Play only, using the URL ("Playstore Link") would make it easier for non-tech-savvy users, though the pure package name would suffice (and can be deduced from the link). It would also make it easier to look it up.

**Comment by rajdedhia10** [comment]
I had made a Pull Request which adds a Package Name column to the csv table but it is currently left on hold until Techlore team decides how to integrate the data with website/app.

**Comment by IzzySoft** [comment]
Great! You could then include a `closes #36` there – so when it gets merged, it will automatically close this issue here and send a note to all watchers, including the information what caused the close (here: the PR being merged) so one can follow up and check. And it would automatically be linked from here – which I can do myself, of course: #17 it is, right?

**Comment by rajdedhia10** [comment]
I have added the `closes #36` tag to my PR as you said. So the issue will be closed when the PR is merged. Thanks.

**Comment by PrestoSole** [comment]
> 
> 
> Great! You could then include a `closes #36` there – so when it gets merged, it will automatically close this issue here and send a note to all watchers, including the information what caused the close (here: the PR being merged) so one can follow up and check. And it would automatically be linked from here – which I can do myself, of course: #17 it is, right?

Hello, how and where do I add the closes#number in my pull request

**Comment by IzzySoft** [comment]
@PrestoSole usually in your initial post (and before it's merged). Guess in the comments it would be ignored, but I've never tested that theory. In the initial post it always worked for me.

**Comment by PrestoSole** [comment]
> @PrestoSole usually in your initial post (and before it's merged). Guess in the comments it would be ignored, but I've never tested that theory. In the initial post it always worked for me.

@IzzySoft Got it, Thanks

**Comment by alex9099** [comment]
This is kinda urgent to merge, as with more application submitted more empty package names we will have.

I agree that the package name is a must, as it should identify the app without any doubt, while the name can be redundant (also language dependent on some apps)

**Comment by techlore-official** [comment]
No reason to merge yet if:
A) It’s one more data point for people to manually type, discouraging entries and making the barrier harder.
B) Manual typing will lead to typos and require more oversight.

This will only be implemented once it can be automated to save everybody time. Longer vision is important—the app should automatically get all of this information to automated the entire process. Once the app is complete, it won’t be hard to fill in the gaps of all the missing applications. Urgency for this is not as dire as it seems.

**Comment by IzzySoft** [comment]
A) can be done by copy-pasting from the URL, to avoid those typos  
B) will be avoided thus :wink:  
C) the more data already there when adding that column, the more work it will be updating existing entries

While I understand your reasoning, and usually do not go "pushing": putting everything together I'd say hurry up automating it then. Adding the packageName automatically *when entering the data* (e.g. via "please paste the URL or packageName" and verifying in the background) is easy compared to filling the new packageName field based on ambiguous app names – so the longer you postpone this, the harder that job will be. And keep in mind that will be a manual job mainly, due to ambiguosity of app names. Would be a pity being thrown back just because of that.

**Comment by henryistaken** [comment]
Good news everyone, package names & version numbers are now found via our new application (https://github.com/techlore/Plexus-app)

We'll be transitioning all the current data & contribution workflow over to the new repo and updating the Plexus website to follow the new data in the coming weeks.
